app/scripts/views/support/create_secure_password.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const View = require('views/base').extend({
+    className: 'create_secure_password',
+    template: require('stache!templates/support/create_secure_password'),
+    afterRender: function () {
+      this.window.document.cookie = 'canGoBack=1; path=' + this.window.location.pathname;
+    }
+  });
+
+  require('cocktail').mixin(
+    View,
+    require('views/mixins/back-mixin')
```

Comment:
Similar to [the comment above](https://github.com/mozilla/fxa-content-server/pull/4014/files/a7f8ae17e5e05e947460798f7aa105a2ad8ff8c8#r73482564) about this being valid, but strange since we don't do it anywhere else.